src/Toggle/toggleStyles.ts

```diff
@@ -2,13 +2,13 @@ import styled from '@emotion/styled';
 import { ToggleProps } from './Toggle';
 import { hexToRgb } from '../styles/stylesUtil';
 
-export const Label = styled.span<Pick<ToggleProps, 'disabled'>>(({ theme, disabled }) => ({
+export const Label = styled.span<Pick<ToggleProps<unknown>, 'disabled'>>(({ theme, disabled }) => ({
```

Comment:
We still have to type these here, but it doesn't matter what it's typed as so I chose unknown.